vizro-core/src/vizro/__init__.py

```diff
@@ -5,6 +5,6 @@
 
 __all__ = ["Vizro"]
 
-__version__ = "0.1.7"
+__version__ = "0.1.8.dev0"
```

Comment:
Did this always come with 0 at the end? If yes, just ignore

```suggestion
__version__ = "0.1.8.dev"
```

Reply:
Yes, it's always `dev0` to begin with.